jumpscale/packages/auth/bottle/auth.py

```diff
@@ -0,0 +1,284 @@
+import base64
+import json
+from functools import wraps
+from urllib.parse import urlencode
+
+import nacl
+import requests
+from beaker.middleware import SessionMiddleware
+from bottle import Bottle, abort, redirect, request, response
+from nacl.public import Box
+from nacl.signing import VerifyKey
+
+from jinja2 import Environment, FileSystemLoader, select_autoescape
+from jumpscale.god import j
+
+SESSION_OPTS = {"session.type": "file", "session.data_dir": "./data", "session.auto": True}
```

Comment:
let's put the session data in some vardir better, otherwise it'll show up in the repo, better we add it to .gitignore and maybe call it session_data or something

Reply:
done, thanks